### ISSUE TITLE
Windows build related bug

### ISSUE THREAD

**Issue by db4** [janestreet/ppx_core#4]
Please replace in `ppx_core/js-utils/gen_install.ml`
```
let remove_cwd =
  let prefix = Sys.getcwd () ^ "/" in
```
with
```
let remove_cwd =
  let prefix = Sys.getcwd () ^ Filename.dir_sep in
```
otherwise build fails under Windows. This applies to all Jane Street modules.

**Comment by diml** [comment]
I'll fix all packages this week

**Comment by db4** [comment]
Please also update the opam packages

**Comment by diml** [comment]
It's fixed in 113.33.03, I moved this script to [js-build-tools](https://github.com/janestreet/js-build-tools)
